src/modules/menu/components/MenuLangSelector.vue

```diff
@@ -8,7 +8,7 @@
 </style>
 
 <script>
-  import LangSwitchToolbar from "@/modules/i18n/components/LangSwitchToolbar";
+  import LangSwitchToolbar from "../../i18n/components/LangSwitchToolbar";
```

Comment:
out of curiosity: to me the `@/modules/...` way of addressing files looks cleaner than the `../../...` way, what's the reason for the change?

Reply:
Some of them where changed because otherwise I couldn't the store in isolation in Cypress. This `@/` notation is handled by Vue and webpack, but if left as is and then imported straight into a `..._spec.js` file from Cypress, Cypress can't handle the `@` part (I read it's possible to configure something somewhere, I also prefer the Vue notation I could give it a look)

Reply:
:+1: if possible

Reply:
So in the end I've separated E2E testing (which doesn't require to get a portion of the app) and Unit testing into two categories of test. This way I could keep the `@` notation (and it made a bit more sense...)